packages/core/src/Checkbox/Checkbox.tsx

```diff
@@ -59,6 +60,10 @@ const CheckBoxWrapper = styled.div`
   }
 
   > input {
+    & ~ svg[data-name='empty'] {
+      color: ${(props) =>
+        props.disabled ? getPaletteColor('border.base')(props) : getPaletteColor(props.unselectedColor)};
```

Comment:
below we set the svg with that attribute to display: none so I don't think this'll work.
```
& ~ svg[data-name='empty'] {
  display: none;
}
```

Additionally, I'd prefer if we just did `color: ${getPaletteColor('base')}` outside of any selectors

Reply:
> 

@pcln-james , the two places with `display: none` are for `> input:indeterminate` and `> input:checked`, I think it will work(it works fine on local). 

When I try out `${getPaletteColor('base')}`, the default color for disabled unchecked checkbox is box, and on hover, it's grey. I am not remotely familiar with the codebase but my guess would be that it could be a breaking change? Not sure (plus I see everywhere in the code is using `getPaletteColor('border.base')(props)` when handling props.disabled